tfjs-backend-webgpu/src/kernels/pool2d_webgpu.ts

```diff
@@ -71,7 +71,7 @@ export class Pool2DProgram implements WebGPUProgram {
         ivec4 coords = getOutputCoords();
         if (all(lessThan(coords, ${getShapeCoords(this.outputShape)}))) {
           int batch = coords[0];
-          ivec2 xRCCorner = coords.yz * stride - pad;
+          ivec2 xRCCorner = coords.yz * stride.yx - pad;
```

Comment:
Rather than changing here, I think you should adjust the value sequence when writing them to uniform. For example, in `AvgPool.ts` and `MaxPool.ts`, should use `convInfo.strideHeight, convInfo.strideWidth` instead of `convInfo.strideWidth, convInfo.strideHeight`. And please also change other values sequence correspondingly to make it correct. You can use Conv2D as a reference.